app/styles/Home.module.css

```diff
@@ -0,0 +1,19 @@
+.container {
+    padding: 0 2rem;
+}
+
+.main {
```

Comment:
These are the default styles from create-next and subject to change. Leaving them in because they are referenced in index.ts and show the next styling structure